xchange-examples/src/main/java/org/knowm/xchange/examples/bitstamp/account/BitstampFiatWithdrawal.java

```diff
@@ -0,0 +1,60 @@
+package org.knowm.xchange.examples.bitstamp.account;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.bitstamp.BitstampAuthenticatedV2.AccountCurrency;
+import org.knowm.xchange.bitstamp.BitstampAuthenticatedV2.BankCurrency;
+import org.knowm.xchange.bitstamp.BitstampAuthenticatedV2.Country;
+import org.knowm.xchange.bitstamp.dto.account.BitstampBalance;
+import org.knowm.xchange.bitstamp.dto.account.BitstampDepositAddress;
+import org.knowm.xchange.bitstamp.service.BitstampAccountServiceRaw;
+import org.knowm.xchange.examples.bitstamp.BitstampDemoUtils;
+import org.knowm.xchange.service.account.AccountService;
+
+public class BitstampFiatWithdrawal {
+
+  public static void main(String[] args) throws IOException {
+
+    Exchange bitstamp = BitstampDemoUtils.createExchange();
+    AccountService accountService = bitstamp.getAccountService();
+
+    raw((BitstampAccountServiceRaw) accountService);
+  }
+
+  private static void raw(BitstampAccountServiceRaw accountService) throws IOException {
+
+    // Get the account information
+    BitstampBalance bitstampBalance = accountService.getBitstampBalance();
+    System.out.println("BitstampBalance: " + bitstampBalance);
+
+    BitstampDepositAddress depositAddress = accountService.getBitstampBitcoinDepositAddress();
+    System.out.println("BitstampDepositAddress address: " + depositAddress);
+
+    accountService.withdrawSepa(
+        new BigDecimal("150"),
+        "Kovin Kostner",
```

Comment:
Can you use some fake data instead here?

Reply:
Done